src/main/java/org/apache/commons/text/lookup/ScriptStringLookup.java

```diff
@@ -69,7 +69,7 @@ public String lookup(final String key) {
         if (key == null) {
             return null;
         }
-        final String[] keys = key.split(SPLIT_STR);
+        final String[] keys = key.split(SPLIT_STR, 2);
```

Comment:
Maybe worth adding a line to the Javadoc, saying that extra `:` symbols are ignored.

Reply:
Done, changed the complete Javadoc as it was not the correct one